collect_stata/read_stata.py

```diff
@@ -0,0 +1,145 @@
+"""read_stata.py"""
+__author__ = "Marius Pahl"
+
+import logging
+import re
+
+import pandas as pd
+
+LOGGER = logging.getLogger(__name__)
+
+
+def cat_values(varscale, data):
+    """
+    Extract categorical metadata from stata files
+
+    Input:
```

Comment:
This section is not in sync with the interface of the function.

Reply:
#18